### PR TITLE
More Tests: Add test for GetAssignmentDetailsServlet.java

### PR DESCRIPTION
**Pull request by rsampdev**

### CODE REVIEW

File: project/src/test/java/com/google/autograder/data/DatabaseTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+        assertEquals(1, entityCountAfterSaving);
+    }
+
+    @Test
+    public void deleteASingleEntity() {
+        int entityCountAfterDeleting = 0;
+        int entityCountBeforeSaving = 0;
+        int entityCountAfterSaving = 0;
+
```
Comment (BenLin00) [inline project/src/test/java/com/google/autograder/data/DatabaseTest.java]: I think we should use a helper function to check entity count as 0

**Review by mahimashanware — COMMENTED** [review]
lgtm

**Review by mahimashanware — DISMISSED** [review]

File: project/src/test/java/com/google/autograder/data/DatabaseTest.java (hunk 1 above)
Comment (jonathanjlin) [reply project/src/test/java/com/google/autograder/data/DatabaseTest.java]: Yes, please see #65 for an example

**Review by jonathanjlin — DISMISSED** [review]

File: project/src/test/java/com/google/autograder/data/DatabaseTest.java (hunk 1 above)
Comment (rsampdev) [reply project/src/test/java/com/google/autograder/data/DatabaseTest.java]: +1

**Review by BenLin00 — DISMISSED** [review]
lgtm

**Review by BenLin00 — APPROVED** [review]

**Pull request merged**